chocolatey/README.md

```diff
@@ -0,0 +1,14 @@
+# nvs chocolatey package
+
+Chocolatey package for [nvs](https://github.com/jasongin/nvs/)
+
+## Test the package locally
+
+To test this package locally you will need to:
+
+1. Install [Chocolatey](https://chocolatey.org/install)
+1. Open an elevated PowerShell window
```

Comment:
Why elevated? Does the installer install the MSI in all-users mode by default? Or does it require elevation for some other reason?

Reply:
I copy pasted the instructions for another project I'm working on and had trouble installing without elevation. If nvs can get installed without admin privileges then I'll remove it.

Reply:
I've tried testing locally without privileges and it fails:

Reply:
The nvs MSI can install either in a per-user location or a per-machine location, depending on the `ALLUSERS` property. For the per-user installation, it does not require elevation. From the error above it looks like Chocolatey requires elevation because it tries to put some files in `C:\ProgramData\chocolatey`. Since I'm not very familiar with Chocolatey I don't know if that's expected or avoidable.

Reply:
From what I've read it is expected. There's an option to install it with non admin privileges that will prevent it but that's not what most users do [Chocolatey is installed](https://chocolatey.org/docs/installation).